### ISSUE TITLE
Incorrect documentation about computing var in BatchNorm1d

### ISSUE THREAD

**Issue by csukuangfj** [pytorch/pytorch#54858]
## 📚 Documentation

The following says
https://github.com/pytorch/pytorch/blob/9e6877c5c5fc4e01640d3f5ed50d7c0b8e52c5aa/torch/nn/modules/batchnorm.py#L195-L196

But the implementation actually computes an **unbiased** estimator:
https://github.com/pytorch/pytorch/blob/9e6877c5c5fc4e01640d3f5ed50d7c0b8e52c5aa/aten/src/ATen/native/Normalization.cpp#L262-L265

**Comment by csukuangfj** [comment]
Turns out there are several places in the documentation that have the same problem

```
$ git grep -n "via the biased estimator"
torch/_torch_docs.py:8197:via the biased estimator. Otherwise, Bessel's correction will be used.
torch/_torch_docs.py:8220:via the biased estimator. Otherwise, Bessel's correction will be used.
torch/_torch_docs.py:8250:via the biased estimator. Otherwise, Bessel's correction will be used.
torch/_torch_docs.py:8273:via the biased estimator. Otherwise, Bessel's correction will be used.
torch/nn/modules/batchnorm.py:196:    via the biased estimator, equivalent to `torch.var(input, unbiased=False)`.
torch/nn/modules/batchnorm.py:303:    via the biased estimator, equivalent to `torch.var(input, unbiased=False)`.
torch/nn/modules/batchnorm.py:410:    via the biased estimator, equivalent to `torch.var(input, unbiased=False)`.
torch/nn/modules/batchnorm.py:519:    The standard-deviation is calculated via the biased estimator, equivalent to
torch/nn/modules/instancenorm.py:75:    The standard-deviation is calculated via the biased estimator, equivalent to
torch/nn/modules/instancenorm.py:156:    The standard-deviation is calculated via the biased estimator, equivalent to
torch/nn/modules/instancenorm.py:230:    The standard-deviation is calculated via the biased estimator, equivalent to
torch/nn/modules/normalization.py:98:    The standard-deviation is calculated via the biased estimator, equivalent to
torch/nn/modules/normalization.py:190:    The standard-deviation is calculated via the biased estimator, equivalent to
```

**Comment by ngimel** [comment]
The implementation computes unbiased estimator for running variance, and biased estimator for actual normalization.

**Comment by csukuangfj** [comment]
Aha, I see. Thanks.
